tailcfg/tailcfg.go

```diff
@@ -1765,18 +1772,6 @@ type Debug struct {
 	Exit *int `json:",omitempty"`
 }
 
-func appendKey(base []byte, prefix string, k [32]byte) []byte {
```

Comment:
This is dead code. I suspect the key types in `tailscale.com/tailcfg` were moved to `tailscale.com/types/key` at some point?

Reply:
yup